### PR TITLE
newdevice: Exterity AvediaPlayer support

### PR DESCRIPTION
**Pull request by FTBZ**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

---------

- Add support for the Exterity AvediaPlayer units.

This a video converter to IP, we have no category for this kind of stuff... I used Appliance.

screen shot 2017-02-02 at 17 09 22
screen shot 2017-02-02 at 17 09 33

### CODE REVIEW

**Comment by laf** [comment]
Unit tests are failing at the moment.

Can you add sysObjectId output to the unit test file as well so it's more complete.

File: includes/definitions/avr-hd.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+os: avr-hd
+text: 'avr-hd'
+type: appliance
+icon: exterity
+over:
+    - { graph: device_bits, text: Traffic }
+discovery:
+    - sysDescrId:
```
Comment (laf) [inline includes/definitions/avr-hd.yaml]: We really should try and use sysObjectId for detection if possible. If not it's sysDescr anyway

**Comment by FTBZ** [comment]
Can you double check the code? I've noticed that all my devices are recognized as avr-hd and I don't know if it's since this PR or not (I've made some try after your message with the sysObjectId).

File: includes/definitions/avr-hd.yaml (hunk 1 above)
Comment (FTBZ) [inline includes/definitions/avr-hd.yaml]: SNMPwalk here: 
http://pastebin.com/raw/Rem5nv9d

Don't think that's possible?

Comment (murrant) [inline includes/definitions/avr-hd.yaml]:
```
Discovery:
    sysObectID:
        - .1.3.6.1.4.1.28194.
```

**Comment by laf** [comment]
I've just pushed some small changes. Will merge once travis clears.

File: includes/definitions/avr-hd.yaml (hunk 1 above)
Comment (FTBZ) [inline includes/definitions/avr-hd.yaml]: Ok thx. During my test I used the latest digit, it was my error.

**Pull request merged**